packages/flutter/test/services/binding_test.dart

```diff
@@ -44,7 +44,7 @@ class TestBinding extends BindingBase with SchedulerBinding, ServicesBinding {
   BinaryMessenger createBinaryMessenger() {
     return super.createBinaryMessenger()
       ..setMockMessageHandler('flutter/assets', (ByteData message) async {
-        if (const StringCodec().decodeMessage(message) == 'LICENSE') {
+        if (const StringCodec().decodeMessage(message) == 'LICENSE' || const StringCodec().decodeMessage(message) == 'NOTICES') {
```

Comment:
nit: add the same TODO here so we don't forget to remove this as well?

Reply:
Done